test/unit/test-cookies.js

```diff
@@ -231,4 +232,13 @@ describes.fakeWin('test-cookies', {amp: true}, (env) => {
     );
     expect(doc.cookie).to.equal('');
   });
+
+  it('can set cookie', () => {
+    expect(canSetCookie(win)).to.be.true;
```

Comment:
just for my FYI, what's the implicit domain?

Reply:
I think it's just localhost for unit tests.